allennlp/data/dataset_readers/quora_paraphrase.py

```diff
@@ -0,0 +1,82 @@
+from typing import Dict
+import re
+import logging
+import csv
+
+from overrides import overrides
+
+from allennlp.common.file_utils import cached_path
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.fields import LabelField, TextField, Field
+from allennlp.data.instance import Instance
+from allennlp.data.tokenizers import Tokenizer, WordTokenizer
+from allennlp.data.tokenizers.word_splitter import JustSpacesWordSplitter
+from allennlp.data.token_indexers import TokenIndexer, SingleIdTokenIndexer
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def parse_file_uri(uri: str):
```

Comment:
I think you don't need this anymore.  (Though, as I said before, I think it's totally reasonable to talk about adding this functionality more generally, in `common.file_utils`.  It should just be in a separate PR.)

Reply:
Oh, right. Let me remove this.